ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionValidator.java

```diff
@@ -217,6 +217,8 @@ public MainnetTransactionValidator(
     // org.bouncycastle.math.ec.ECCurve.AbstractFp.decompressPoint throws an
     // IllegalArgumentException for "Invalid point compression" for bad signatures.
     try {
+      // TODO: this is where we are checking the signature. We have to fix the v value if this is a
```

Comment:
This TODO isn't related to this change. We should remove it.

Reply:
Yup!